transformers_interpret/explainers/sequence_classification.py

```diff
@@ -50,12 +48,14 @@ def __init__(
         model: PreTrainedModel,
         tokenizer: PreTrainedTokenizer,
         attribution_type: str = "lig",
+        custom_labels: Optional[List[str]] = None
     ):
         """
         Args:
             model (PreTrainedModel): Pretrained huggingface Sequence Classification model.
             tokenizer (PreTrainedTokenizer): Pretrained huggingface tokenizer
             attribution_type (str, optional): The attribution method to calculate on. Defaults to "lig".
+            custom_labels (List[str], optional): To use custom labels instead Pretrained Model configured
```

Comment:
I'd like to expand a little bit here on the description of what this does. I'm thinking 

`custom_labels (List[str], optional):  Applies custom labels to label2id and id2label configs. Labels must be same length as the base model configs' labels. Labels and ids are applied index-wise. Defaults to None.`